rpc/src/module/chain.rs

```diff
@@ -251,4 +254,17 @@ impl<CS: ChainStore + 'static> ChainRpc for ChainRpcImpl<CS> {
     fn get_tip_block_number(&self) -> Result<BlockNumber> {
         self.get_tip_header().map(|h| h.inner.number)
     }
+
+    fn get_block_finalized_reward_info(&self, hash: H256) -> Result<Option<BlockRewardView>> {
+        Ok(self
+            .shared
+            .store()
+            .get_block_header(&hash)
+            .and_then(|header| {
+                self.shared
+                    .finalize_block_reward(&header)
```

Comment:
I'm afraid `finalize_block_reward` requires parent hash instead of the current block hash.

Reply:
The reason is that the info is required when constructing the new block, which hash is not available yet.

Reply:
Maybe we should rename `finalize_block_reward` to more verbose `finalize_block_reward_via_parent_hash`, and `RewardCalculator::block_reward` to `RewardCalculator::block_reward_via_parent_hash`.

Reply:
`fn finalize_block_reward(&self, _parent: &Header)`  implicitly tell that.

Reply:
updated, would find the parent first and then finalize_block_reward.